src/Agent.Listener/Configuration/WindowsServiceControlManager.cs

```diff
@@ -142,6 +142,7 @@ private void GrantDirectoryPermissionForAccount(string accountName)
             string workFolder = IOUtil.GetWorkPath(HostContext);
             Directory.CreateDirectory(workFolder);
             Trace.Info(StringUtil.Format("Set full access control to group for the folder {0}", workFolder));
+            Trace.Info(StringUtil.Loc("GrantingFilePermissions"));
```

Comment:
I think we would like do `Terminal.write` instead of `Trace` from the caller. :)